libs/io_service/tests/performance/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2019 The STE||AR-Group
+#
```

Comment:
Empty tests CMakeLists can probably be removed

Reply:
Yes, we should probably do that... I won't do it here as we've kept them for other modules and that keeps the modules' `tests/CMakeLists.txt` pretty much the same for now.